### PR TITLE
Tom now rejects reviews based on banned e-mails and is less spammy

### PR DESCRIPTION
**Pull request by olehermanse**

### CODE REVIEW

File: tom/bot.py [hunk 1]
```diff
@@ -24,6 +24,8 @@ def __init__(self, config, secrets, directory, interactive):
         self.default_maintainers = config["reviewers"]
         self.trusted = config["trusted"]
         self.jenkins_repos = config["jenkins_repos"]
+        banned_emails = config.get("banned_emails", {})
+        self.banned_emails = [v for _, v in banned_emails.items()]
```
Comment (vpodzime) [inline tom/bot.py]: `for v in banned_emails.values()` would be nicer

Comment (olehermanse) [reply tom/bot.py]: Fixed, thanks!

File: tom/bot.py [hunk 2]
```diff
@@ -84,19 +91,42 @@ def ping_reviewer(self, pr):
...
+        bad_emails = set()
+        for email in pr.emails:
+            hash = email_sha256(email)
+            log.debug("{}: {}".format(email, hash))
+            if email in self.banned_emails or hash in self.banned_emails:
+                log.info("Found banned email: " + email)
+                bad_emails.add(email)
+
+        obfuscated = [f"{e[0]}***@{e[e.index('@')+1:]}" for e in bad_emails]
+        bad_emails = ",".join(obfuscated)
```
Comment (karlhto) [inline tom/bot.py]: What about `", "` for slightly easier readability? :^)

Comment (olehermanse) [reply tom/bot.py]: Good suggestion, feel free to open another PR! :)

**Pull request merged**